test/hotspot/jtreg/gtest/WindowsProcessorGroups.java

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (c) 2020, 2021, Oracle and/or its affiliates. All rights reserved.
```

Comment:
NIT: Copyright year is 2024 only. I think you can delete the (next) line about SAP as well.